math/fibonacci/fibonacci.go

```diff
@@ -45,3 +45,18 @@ func Formula(n uint) uint {
 	powPhi := math.Pow(phi, float64(n))
 	return uint(powPhi/sqrt5 + 0.5)
 }
+
+func ForSlice(n uint) uint {
+
+	if n == 0 {
+		return 0
+	}
+	k := make([]uint, n+1)
```

Comment:
If you're making a slice here and not using it in any meaningful way, what is the point of wasting all that memory? It would be much better to just have three variables without wasting memory.

Reply:
Thank you for the valuable information ☺